sources/models/shared-defaults/metal-host-containers.toml

```
@@ -0,0 +1,14 @@
+# Both containers are disabled by default in metal variants because the user
+# must supply user data in order to use the containers.  The admin container
+# isn't useful without SSH keys/CA certs, and the control container can only be
+# used with hybrid SSM off of AWS.  Metal variant users might not want to use
+# either of those options.
+[settings.host-containers.admin]
+enabled = false
+superpowered = true
+source = "public.ecr.aws/bottlerocket/bottlerocket-admin:v0.7.2"
+
+[settings.host-containers.control]
+enabled = false
+superpowered = false
+source = "public.ecr.aws/bottlerocket/bottlerocket-control:v0.5.2"
```

Need to make sure these are the latest so we don't end up with different versions for migrations to consider:

```suggestion
[settings.host-containers.admin]
enabled = false
superpowered = true
source = "public.ecr.aws/bottlerocket/bottlerocket-admin:v0.7.3"

[settings.host-containers.control]
enabled = false
superpowered = false
source = "public.ecr.aws/bottlerocket/bottlerocket-control:v0.5.4"
```

Alternatively we might want to rename the file "public-host-containers.toml" so we don't multiply the number of references to the same images.